recipe/meta.yaml

```diff
@@ -29,11 +38,12 @@ test:
   requires:
     - tree  # [linux]
```

Comment:
```suggestion
  requires:
    - tree  # [linux]
    - cmake
```

You'll want to add a test CMakeLists.txt that does something like

```cmake
find_package(mathdx REQUIRED COMPONENTS cufftdx cublasdx)
```

Reply:
You could use a jinja variable to depend on the oldest supported version of cutlass in the test environment, then the test would fail during CMake config if mathdx requires something newer.